roles/openshift_metering/tasks/install.yaml

```diff
@@ -0,0 +1,93 @@
+---
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-metering-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Ensures {{ mktemp.stdout }}/metering-manifests dir exists
+  file:
+    path: "{{ mktemp.stdout }}/metering-manifests"
+    state: directory
+
+- name: Copy files to temp directory
+  copy:
+    src: "{{ item }}"
+    dest: "{{ mktemp.stdout }}/metering-manifests/{{ item | basename }}"
+  with_fileglob:
+  - "*.yaml"
+
+- name: Copy admin client config
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{ mktemp.stdout }}/admin.kubeconfig
+  changed_when: false
+
+- name: Update Metering Helm Operator deployment image
```

Comment:
Consider making the original yaml file a jinja template and using the `template` task instead here?

Reply:
Atm I'm trying to avoid unnecessary divergence from the upstream repo. The files are already templates (in a different form) in the upstream repo, and these are the rendered out results. Basically, the only things we need to make configurable for this are a few fields in the YAML which we can just replace. I asked @smarterclayton about how I should go about it and he recommended I went down this path.